7.0-cli/bin/docker-environment

```diff
@@ -31,11 +31,24 @@ fi
 mkdir -p $MAGENTO_ROOT
 chown www-data:www-data $MAGENTO_ROOT
 
+CRON_LOG=/var/log/cron.log
+
+# Setup Magento cron
+echo "* * * * * www-data /usr/bin/php ${MAGENTO_ROOT}/bin/magento cron:run | grep -v \"Ran jobs by schedule\" >> ${MAGENTO_ROOT}/var/log/magento.cron.log" > /etc/cron.d/magento
```

Comment:
Hrm, it must have been at the time of this merge, as the cron was successfully running. I agree that it should probably be `/use/local/bin/php` though.